src/overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_scope.h

```diff
@@ -385,7 +287,7 @@ typedef enum {
 #define PAUSE_PROMPT_YES 0
 #define PAUSE_PROMPT_NO 4
 
-typedef enum {
+typedef enum PromptQuad {
```

Comment:
just a question in passing, are you now naming all types instead of keeping typedef-ed anonymous types?

Reply:
We have been slowly moving to avoid anonymous types. One of the reasons is because tools work better with named types (compiler warnings, intellisense, gdb, etc)